hrjobcontract/xml/organisation_contact_install.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
```

Comment:
Now might be a good time to move the xml files for option groups into their own folder. i.e create an option_group folder within the xml folder and move the xml option groups related files there. I took a quick look and it seems the files are referenced in one or two places, so shouldn't be a big change.

Reply:
Alright. I actually did that first but reverted to maintain conformity with previous implementations. 👍